apps/src/music/views/MusicView.jsx

```diff
@@ -237,7 +239,10 @@ class UnconnectedMusicView extends React.Component {
       );
     }
 
-    if (prevProps.currentLevelIndex !== this.props.currentLevelIndex) {
+    if (
+      prevProps.currentLevelIndex !== this.props.currentLevelIndex &&
+      this.props.isActive
```

Comment:
if we switch to conditional rendering do we need to worry about `isActive`?

Reply:
We didn't need it in the end.